<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1425</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QLabel" name="background_label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>1320</width>
         <height>700</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 20px;
background-image: url(:/images/images/las.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>1320</width>
         <height>700</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.715909, stop:0 rgba(0,0,0,9), stop:0.375 rgba(0,0,0,50), stop:0.835227 rgba(0,0,0,75));
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>1220</width>
         <height>650</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,100);
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>190</y>
         <width>331</width>
         <height>441</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="login_button">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>197</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI Semilight</family>
          <pointsize>15</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(170,170,170), stop:1 rgba(85, 112, 98, 219));
border-radius: 10px;
	color: rgb(0, 0, 0);
}

QPushButton:hover {background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(85,112,98,219), stop:1 rgb(170,170,170));
}

QPushButton:pressed {
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(105,118,132,200);
}</string>
        </property>
        <property name="text">
         <string>Log in</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI Semilight</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(179, 179, 179);</string>
        </property>
        <property name="text">
         <string>Already have a account?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>1201</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Nirmala UI Semilight</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(179, 179, 179);
</string>
       </property>
       <property name="text">
        <string>Strona Logowania idk jaki napis</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>190</y>
         <width>331</width>
         <height>441</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="create_account_button">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>137</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI Semilight</family>
          <pointsize>15</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(170,170,170), stop:1 rgba(85, 112, 98, 219));
border-radius: 10px;
	color: rgb(0, 0, 0);
}

QPushButton:hover {background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(85,112,98,219), stop:1 rgb(170,170,170));
}

QPushButton:pressed {
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(105,118,132,200);
}</string>
        </property>
        <property name="text">
         <string>Create account</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>110</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI Semilight</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(179, 179, 179);</string>
        </property>
        <property name="text">
         <string>Don't have an account?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="create_account_button_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI Semilight</family>
          <pointsize>15</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(170,170,170), stop:1 rgba(85, 112, 98, 219));
border-radius: 10px;
	color: rgb(0, 0, 0);
}

QPushButton:hover {background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(85,112,98,219), stop:1 rgb(170,170,170));
}

QPushButton:pressed {
	padding-left: 5px;
	padding-top: 5px;
	background-color: rgba(105,118,132,200);
}</string>
        </property>
        <property name="text">
         <string>Complete the sessions</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>223</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI Semilight</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(179, 179, 179);</string>
        </property>
        <property name="text">
         <string>Don't have an account?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
